doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,391 @@
+  set grid arguments     = 3,1,1 : -0.010,-0.01,-0.01 : 0.035, 0.01,0.01 : true
+  set initial refinement = 6
+  end
+  
+  
+Boundaries Conditions
+~~~~~~~~~~~~~~~~~~~~~
+
+The bottom of the domain is set as an outlet to allow the particles to leave the simulation domain, thus reducing the computational time of the simulation.
+
+.. code-block:: text
+    
+  subsection DEM boundary conditions
+  set number of boundary conditions = 1
+
+  subsection boundary condition 0
+  set boundary id         = 0
+  set type                = outlet
+  end
+  end
+
+
+Lagrangian Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The particles have a polydisperse size distribution with a density of 1922 :math:`kg/m^3`. 400 000 particles would be needed to match the 2 g of sand used in the experiment. However, to reduce the simulation's duration of this example, we insert only 200 000 particles.
+
+According to Hu *et al.* [#hu2022]_ the rolling friction and the surface energy parameters impact the final simulated angle of repose the most. Thus, we simulate twelve combinations of those parameters. The ``rolling friction`` is between 0.3 and 0.7, and the ``surface energy`` is between 0.0010 and 0.0100. The ``granuheap.prm`` file of this example is specifically for a ``rolling friction`` of 0.5 and a ``surface energy`` of 0.0100. 
+
+The water volume fraction of the wet sand used in the experiment is 0.325 %. According to the work of Liefferink *et al.* [#Liefferink2018]_, this water volume fraction matches an ``friction coefficient`` of 0.5. 
+
+.. code-block:: text
+
+  subsection lagrangian physical properties
+  set g                        = -9.81, 0, 0
+  set number of particle types = 1
+  subsection particle type 0
+  set size distribution type            = custom
+  set custom diameters                  = 1.02e-4 , 1.16e-4 , 1.33e-4 , 1.52e-4 , 1.75e-4 , 2.00e-4 , 2.29e-4 , 2.62e-4 , 3.01e-4 , 3.44e-4
+  set custom volume fractions           = 0.04 , 0.06 , 0.07 , 0.10 , 0.13 , 0.15 , 0.16 , 0.14 , 0.10 , 0.05
+  set number of particles               = 200000
+  set density particles                 = 1922
+  set young modulus particles           = 5.94e4
+  set poisson ratio particles           = 0.4
+  set restitution coefficient particles = 0.9
+  set friction coefficient particles    = 0.5
+  set rolling friction particles        = 0.5
+  set surface energy particles          = 0.01
+  end
+  set young modulus wall           = 1e7
+  set poisson ratio wall           = 0.33
+  set restitution coefficient wall = 0.9
+  set friction coefficient wall    = 0.5
+  set rolling friction wall        = 0.5
+  set surface energy wall          = 1e-4
+  end
+
+
+
+Model Parameters
+~~~~~~~~~~~~~~~~~~~~
+
+The JKR contact model is used in this case because of the cohesive aspect of wet sand. Indeed, as Hu *et al.* [#hu2022]_ explain, the JKR method can better simulate agglomeration between particles than, for example, the Hertz-Mindlin contact model.
```

Comment:
It is perfect that you justify the use of the JKR force model, but I'm not sure if your justification is right. 

Clearly, the JRK model is better than the Hertz-Mindlin model to simulate particle agglomeration since the JKR model is a cohesive force model and H-M is not. 

Why did you use the JKR model and instead of the DMT for example? (You don't need to answer this question here.) That would be a better justification IMO. 

You could say: We use the JKR model because it has be shown to correctly model particle agglomeration for wet sand (source here).